src/md/community.md

```diff
@@ -1,6 +1,6 @@
-# Community
+# <a name="community"></a>Community
```

Comment:
Why not use the anchor that is generated during the html generation instead of adding this here?